TMDbLibTests/ClientCreditTests.cs

```diff
@@ -33,6 +33,14 @@ public void TestGetCreditTv()
             Assert.Equal("", result.Media.Character);
         }
 
+        [Fact]
+        public void TestMissingCredit()
+        {
+            Credit result = Config.Client.GetCreditsAsync("9999999999").Result;
```

Comment:
We should make a single test that tests the exception-throwing part. Testing each "area" of TMDb once for not-found is probably a good idea. That API hasn't really shined in concistency :)